### DISCUSSION TITLE
Welcome to PHP-Calendar Discussions!

### DISCUSSION THREAD

**Discussion by sproctor** [sproctor/php-calendar#194] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanded001** [comment]
Hi Sean. Been using the calendar for ages now. just lovely. Wondering if by any chance a birthday option is upcoming? :)

**Comment by keithwilson** [comment]
How would I go about changing the current date background to red vs the current yellow? Thanks...

**Reply by keithwilson** [reply]
Never mind, ..... I figured it out as shown below. Maybe someone else has the same 'problem' ?

<style>
/* Modification of calendar.php current date color......
   I added this stylesheet to set today's date in blue. Note the use of !important to override the css file from 
   ajax.googleapis.com/ajax/libs/jquery/$jquery_version/jquery$jq_min.js further on down in this calendar.php file (approximately at Line 
   900 or so).
   **** It will make today's date background in blue, regardless of the theme that you are using.****
1.) Created a file to replace "ui-bg_highlight-soft_25_ffef8f_1x100.png" with a local file named "current-date-background.png",
2.) Then created an images folder in my calendar root, and put my new color file "current-date-background.png" into that folder.
    (The original image was 1px wide x 100px high. My replacement file is 4px wide x 100px high.)
*/

.ui-state-highlight,.ui-widget-content .ui-state-highlight,.ui-widget-header .ui-state-highlight {
    border: 3px solid #3366ff !important; /* was orignally 1px solid #f9dd34 !important; */
				/* line below is what it used to be when accessing the stylesheet online from the above url
				   background: #ffef8f url("images/ui-bg_highlight-soft_25_ffef8f_1x100.png") 50% top repeat-x;
				*/
    background: #ffef8f url("images/current-date-background.png") 50% top repeat-x !important;
    color: #363636 !important; ``
}
</style>
